Fix permissions on /etc/openstack-dashboard for nginx startup

jprorama opened the pull request:
Fixes #2

jprorama committed:
Fix permissions on /etc/openstack-dashboard for nginx startup

flakrat commented:
Looking at the process tree on the os-controllers, there aren't any running under apache. I suspect that the bug is that Bright assigned the wrong group to the directory and files `s/apache/nginx/`

```shell
drwxr-x--- 3 root apache 4096 Dec  9 12:48 /cm/images/os-controller/etc/openstack-dashboard
...
-rw-r----- 1 root apache  7788 Oct 22 15:09 /cm/images/os-controller/etc/openstack-dashboard/cinder_policy.json
-rw-r----- 1 root apache  1388 Oct 22 15:09 /cm/images/os-controller/etc/openstack-dashboard/glance_policy.json
-rw-r----- 1 root apache 10144 Oct 22 15:09 /cm/images/os-controller/etc/openstack-dashboard/keystone_policy.json
-rw-r----- 1 root apache 11114 Oct 22 15:09 /cm/images/os-controller/etc/openstack-dashboard/neutron_policy.json
-rw-r----- 1 root apache 11269 Oct 22 15:09 /cm/images/os-controller/etc/openstack-dashboard/nova_policy.json
```

jprorama committed:
Correct permission change to use nginx group owner

The appropriate fix for the files was to set the process
group owner to nginx rather than opening the file permission.
